.github/workflows/1_2_b_bump_extension_only.yml

```diff
@@ -0,0 +1,73 @@
+name: Bump versions for extension only
```

Comment:
Might be good to have a comment here what this workflow is used for and why it is separate from the others.